projects/admin/src/app/deposit/editor/editor.component.html

```diff
@@ -48,29 +77,63 @@
           </div>
         </div>
       </div>
+      <div class="col-sm-4 text-left text-sm-right">
+        <div class="btn-group" role="group" aria-label="Basic example">
+          <button
+            type="button"
+            class="btn"
+            [ngClass]="{
+              'btn-primary': view === viewItem.key,
+              'btn-light': view !== viewItem.key
+            }"
+            *ngFor="
+              let viewItem of [
+                { key: 'form', icon: 'tasks' },
+                { key: 'preview', icon: 'desktop' },
+                { key: 'json', icon: 'code' }
+              ]
+            "
+            (click)="view = viewItem.key"
+          >
+            <i [class]="'fa fa-' + viewItem.icon"></i>
```

Comment:
class or ngClass?

Reply:
I think it's right to use class binding instead of ngClass because the class name is dynamic.